app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(4, 2) == [4 // 2] * 2
```

Comment:
This test is well written, but again, it would be beneficial to test with a range of numbers to ensure your function can handle different inputs.